src/NuGet.Core/NuGet.Protocol/HttpSource/StsAuthenticationHandler.cs

```diff
@@ -99,7 +99,9 @@ protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage
                 {
                     try
                     {
+#pragma warning disable CA2016 // Forward the 'CancellationToken' parameter to methods
```

Comment:
Adding the suppression in the GlobalSuppressions.cs file didn't resolve the warning. Hence added a suppression directly in the code.

I am planning to create a follow-up issue for us to address CA2016 because it is related to reliability. We fixed a bunch of errors previously in https://github.com/NuGet/NuGet.Client/pull/4944.